### DISCUSSION TITLE
deckmaster with arch linux

### DISCUSSION THREAD

**Discussion by insidesources** [muesli/deckmaster#36] [Q&A] [answered]
Hello!

I came across your deckmaster project and wanted to give it a try,

I git cloned the aur link and built the package, i can run deckmaster from the terminal and it says this:

2021/06/07 15:38:09 Loading deck: /home/meta/Downloads/deckmaster/deckmaster.deck
2021/06/07 15:38:09 open /home/meta/Downloads/deckmaster/deckmaster.deck: no such file or directory

So i saw your github page and it says to make the 99-streamdeck.rules file, i attempted to use that file name, reload the rules and then i also tried to rename it to deckmaster.deck, reload the rules and same output.

Are there general installation guidelines after building the package from the aur? I also don't see any UI for this after building the package.

**Comment by muesli** [comment, marked as answer]
> I git cloned the aur link and built the package, i can run deckmaster from the terminal and it says this:
> 
> 2021/06/07 15:38:09 Loading deck: /home/meta/Downloads/deckmaster/deckmaster.deck
> 2021/06/07 15:38:09 open /home/meta/Downloads/deckmaster/deckmaster.deck: no such file or directory

You're supposed to provide a deck configuration file as a parameter to deckmaster. You can find simple examples in the repo's `decks` directory.

Try starting deckmaster like this:

```bash
./deckmaster -deck decks/main.deck
```

> So i saw your github page and it says to make the 99-streamdeck.rules file, i attempted to use that file name, reload the rules and then i also tried to rename it to deckmaster.deck, reload the rules and same output.

Rename it back to `99-streamdeck.rules`, make sure the content matches the README, and reload the rules once more. You can then just start deckmaster manually from the terminal and skip the systemd service setup. This is something you can still do eventually, should you want to start deckmaster automatically on login.

> Are there general installation guidelines after building the package from the aur? I also don't see any UI for this after building the package.

There's no UI as of yet, the configuration is done via simple toml config files. The examples in the repo are fairly self-explanatory, but I'm happy to answer all your questions, should you run into issues with it!

Hope this helps you getting it running!
